src/ui/map.js

```diff
@@ -44,6 +44,7 @@ import type DragPanHandler, {DragPanOptions} from './handler/drag_pan';
 import type KeyboardHandler from './handler/keyboard';
 import type DoubleClickZoomHandler from './handler/dblclick_zoom';
 import type TouchZoomRotateHandler from './handler/touch_zoom_rotate';
+import defaultLocale from './localization/default_locale';
```

Comment:
Nit: do we need to introduce a new folder if it will only contain a single file? Maybe worth moving to just `src/ui/default_locale.js`.